jsapp/js/components/common/modal.scss

```diff
@@ -415,6 +423,11 @@ $z-modal-x: 10;
         margin-right: 12px;
       }
 
+      .table-media-preview__text {
+        text-align: left;
+        padding: 0 32px 32px 32px;
```

Comment:
We should either force wrap long words, or add a right padding that works, see:

<img width="658" alt="Screenshot 2021-08-18 at 23 17 49" src="https://user-images.githubusercontent.com/2521888/129979518-e706a7f2-4f51-4771-ae89-9e49027bd7b6.png">

Reply:
I was wary of bad breaks but I didn't know that your browser would auto-hyphen words it thinks belong to the dictionary. Even some of the longest words in English are hyphened by the browser:
![image](https://user-images.githubusercontent.com/16762675/130106533-14f63528-e2f9-475a-a6c4-0fdddaebe7d9.png)

Hopefully not many submissions have 1000+ character words :P If the submission contains a typo such as a long sentence without spaces it would just arbitrarily cut the 'word' though